src/include/duckdb/function/scalar/string_functions.hpp

```diff
@@ -85,4 +88,21 @@ struct RegexpMatchesBindData : public FunctionData {
 	unique_ptr<FunctionData> Copy() override;
 };
 
+enum class StringFunctionsType : uint8_t {
+    REVERSE,
```

Comment:
The StringFunctionsType seems a bit excessive here, especially as only PREFIX/SUFFIX/CONTAINS are actually needed. Could you perhaps remove this type?